docs/project_management/pitch.md

```diff
@@ -11,4 +11,15 @@ At the start of each semester, project leads that are looking for members can pi
     - "Graphic design experience preferred"
 - Repository link *if existing project*
 
-?> You do not have to pitch a project if you are working solo or have already found team members interested in joining.
\ No newline at end of file
+?> You will be directed where to submit your pitch before Pitch Days begin. Be sure to get them started as soon as possible.
```

Comment:
I'm not sure I understand the `?>`. Perhaps I'm missing something?

Reply:
We ~~use~~ inherited Docsify for the Handbook, and it provides a shorthand for alerts `!>` and info messages `?>`!

Reply:
Oh that's neat, I like how it looks in the generated docs, yea.